content/departments/product-engineering/strategy-goals/index.md

```diff
@@ -55,7 +57,24 @@ One difference between this year and last is that we anticipate more competition
 
 For those particularly interested in understanding more about competitive frameworks as they apply to Sourcegraph, there is also [this internal document](https://docs.google.com/document/d/1BahpQNdVtg2guhoZ9Jvv0yhhENIQm5dTNjsKMCc7r4k/edit#heading=h.9dsgkokut1e1).
 
-## See also
+### See also
 
 - [Company strategy for FY23](../../../strategy-goals/strategy/index.md#this-year-fy23)
 - [Product/Engineering OKRs for Q1 FY23](https://docs.google.com/presentation/d/1KUOElUkrH-29teXmZBmgmIgLngf-I_6Ikixub1SR0yM/edit#slide=id.g1014ebc164b_0_137) (internal only)
+
+## FY23 Q2 focus
+
+**We’re building for enterprise and strategic accounts**
+We’re building a product that adds value for enterprise (501 - 10k employees) and strategic (10k+ employees) customers because this group has the biggest impact on our revenue and this allows us to get Sourcegraph into the hands of thousands of developers.
+
+**Improving quality is essential to our success**
+To meet the high expectations of all customers, especially enterprise and strategic customers, we need to improve the quality of the end-to-end customer experience of Sourcegraph by investing in areas that will help us reduce the number of defects in releases, reduce the number of incidents/issues, and reduce the time to resolution of incidents/issues.
+
+**We need to improve our deployments to meet customer demand**
+We have a great platform and our customers are struggling with getting to a point where they can use it through a trial or production instance in a reasonable amount of time. We need to ensure that deployments are as seamless as possible and don’t get in the way of customer value. In addition, we continue to see increased interest in Sourcegraph managed deployments since they provide a way to avoid the burden of self-hosting and maintaining Sourcegraph, so we need to quickly scale our ability to support more managed instances to meet these demands. From our customer perspective, managed instances are a Cloud SaaS offering of our product and we also need to think of it in this way. Over time (~2 years), we expect automated managed instances to evolve into a streamlined Sourcegraph hosted self-service solution.
```

Comment:
Would it make sense to drop ```We have a great platform``` in this context - the latter part of the sentence suggests customers are struggling to use it?

Reply:
```suggestion
We have a great product and our customers are struggling with getting to a point where they can use it through a trial or production instance in a reasonable amount of time. We need to ensure that deployments are as seamless as possible and don’t get in the way of customer value. In addition, we continue to see increased interest in Sourcegraph managed deployments since they provide a way to avoid the burden of self-hosting and maintaining Sourcegraph, so we need to quickly scale our ability to support more managed instances to meet these demands. From our customer perspective, managed instances are a Cloud SaaS offering of our product and we also need to think of it in this way. Over time (~2 years), we expect automated managed instances to evolve into a streamlined Sourcegraph hosted self-service solution.
```

Updated to distinguish the product, but that the pain point is around the setup/configuration.